.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Want a git challenge? If you ever accidentally track your node_modules, see if you can go back in git-time such that they were _never tracked in the first place!_

I might accomplish this using `git reset [ sha of initial commit ]` , adding this to node modules, and then recreating your commits using `git add patch`